src/NServiceBus.Gateway.PersistenceTests/NServiceBusGatewayDeduplicationStorageTest.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace NServiceBus.Gateway.PersistenceTests
+{
+    using NUnit.Framework;
+
+    [TestFixture]
+    abstract class NServiceBusGatewayDeduplicationStorageTest
```

Comment:
Aren't we usually making those classes public?

Example

https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.TransportTests/NServiceBusTransportTest.cs#L17
https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.AcceptanceTests/NServiceBusAcceptanceTest.cs#L13